### PR TITLE
2-months blog post and updated info for boat tour and social event

### PR DESCRIPTION
**Pull request by thatdocslady**
All the updates. Please review :)

### CODE REVIEW

**Comment by plaindocs** [comment]
I briefly checked this, didn't run it locally. Seems fine to me.

File: docs/conf/eu/2017/news/announcing-events-volunteers.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+- **Writing Day and Welcome Reception, Sunday September 10th**. We'll be soft-launching the conference with a `Writing Day <http://www.writethedocs.org/conf/eu/2017/writing-day/>`_  and Welcome Reception before we dive into the main sessions. This is an excellent opportunity to register early, meet your fellow documentarians, learn about and work on interesting documentation projects, and make yourself at home!
+
+- **Lightning Talks.** One of our favorite traditions, `lightning talks <http://www.writethedocs.org/conf/eu/2017/lightning-talks/>`_ are as entertaining as they are informative. Each block of lightning talks will have 5-6 spots of up to 5 minutes each, both blocks will be after lunch during the main conference days. Sign-up on the mornings of the main conference days and prepare for some post-lunch fun.
```
Comment (mxsasha) [inline docs/conf/eu/2017/news/announcing-events-volunteers.rst]: Should this be "sign up on the mornings" ?

File: docs/conf/eu/2017/news/announcing-events-volunteers.rst [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+- **Unconference.** Parallel to the afternoon talks, you're welcome to join the `unconference sessions <http://www.writethedocs.org/conf/eu/2017/unconference/>`_. These freestyle sessions are a great opportunity to brainstorm with your fellow documentarians about anything from the latest tooling, the best way to write an email, or how to handle stress and burnout. We have a few great anchor sessions to get your started, but the main sign-up for the unconferene slots will be during the conference itself.
```
Comment (mxsasha) [inline docs/conf/eu/2017/news/announcing-events-volunteers.rst]: unconferene -> unconference

**Review by mxsasha — CHANGES_REQUESTED** [review]

File: docs/conf/eu/2017/news/announcing-events-volunteers.rst [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+The conference is now just under two months away, and we hope you’re getting geared up for an exciting Write the Docs experience in beautiful Prague, Czech Republic.
+
+`Tickets <http://www.writethedocs.org/conf/eu/2017/tickets>`_ are selling quickly – thanks to everyone who has already purchased theirs! We do expect the conference to sell out fully again this year, so hop off that fence you might be sitting on!
+
+If you're still debating whether to attend, check out the `Sessions <http://www.writethedocs.org/conf/eu/2017/speakers/>`_ page for the full list of presentations.
+
+Events and Activities
+---------------------
+
+As always, Write the Docs events are more than just talks. Every year we try to maximize our attendees experience and invest in the social experience of being a part of our community.
```
Comment (ericholscher) [inline docs/conf/eu/2017/news/announcing-events-volunteers.rst]: 2x experience? Maybe: "Every year we try to maximize our attendees enjoyment and invest in the social experience of being a part of our community"

File: docs/conf/eu/2017/news/announcing-events-volunteers.rst [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+Check out what we have in store for you this year in Prague:
+
+- **Boat Tour of Prague, Saturday September 9th**. If you're in town early (or if you're not sure whether to come early), you're invited to join us for a 2-hour cruise on the Vltava river, with a relaxed guided tour in English, light snacks, and light drinks included. This activity is priced separately and isn't included with the conference ticket. You can find all the details in our `Boat Tour <http://www.writethedocs.org/conf/eu/2017/boat/>`_ page, or register directly on our `ticket Website <https://ti.to/writethedocs/write-the-docs-eu-2017>`_.
```
Comment (ericholscher) [inline docs/conf/eu/2017/news/announcing-events-volunteers.rst]: The `(or if you're not sure whether to come early)` confuses the sentence a bit, and could probably be dropped, or made the primary part "If you're deciding if you want to come a day early"..

File: docs/conf/eu/2017/news/announcing-events-volunteers.rst [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+- **Writing Day and Welcome Reception, Sunday September 10th**. We'll be soft-launching the conference with a `Writing Day <http://www.writethedocs.org/conf/eu/2017/writing-day/>`_  and Welcome Reception before we dive into the main sessions. This is an excellent opportunity to register early, meet your fellow documentarians, learn about and work on interesting documentation projects, and make yourself at home!
```
Comment (ericholscher) [inline docs/conf/eu/2017/news/announcing-events-volunteers.rst]: soft-launching? Not sure what this means.

**Review by ericholscher — COMMENTED** [review]
LGTM over than these small nits.

File: docs/conf/eu/2017/news/announcing-events-volunteers.rst (hunk 3 above)
Comment (ericholscher) [reply docs/conf/eu/2017/news/announcing-events-volunteers.rst]: That still reads a bit passive voice. Needs less abstraction I think. "Every year we make sure to hold lots of social events to allow people to connect and form a community", or something.

Comment (plaindocs) [reply docs/conf/eu/2017/news/announcing-events-volunteers.rst]: > As always, Write the Docs events are more than just talks. We've a variety of events where you can meet the rest of the documentarian community. 

still not a huge fan. But simple is better here I think

File: docs/conf/eu/2017/news/announcing-events-volunteers.rst (hunk 4 above)
Comment (plaindocs) [reply docs/conf/eu/2017/news/announcing-events-volunteers.rst]: Come to Prague early, and join us....

File: docs/conf/eu/2017/news/index.rst [hunk 6]
```diff
@@ -10,6 +10,7 @@ More here as it happens!
 .. toctree::
    :glob:
 
+   announcing-events-volunteers
```
Comment (plaindocs) [inline docs/conf/eu/2017/news/index.rst:13]: @thatdocslady sadly we gotta do this, if you just glob the files like it was before, the order doesn't work.

Comment (thatdocslady) [reply docs/conf/eu/2017/news/index.rst:13]: ahhhh I forgot, thanks

File: docs/conf/eu/2017/news/announcing-events-volunteers.rst [hunk 7]
```diff
@@ -15,17 +15,17 @@ If you're still debating whether to attend, check out the `Sessions <http://www.
 Events and Activities
 ---------------------
 
-As always, Write the Docs events are more than just talks. Every year we try to maximize our attendees experience and invest in the social experience of being a part of our community.
+As always, Write the Docs events are more than just talks. Every year we offer a wider range of activities that give our community members different ways to socialize, collaborate, and connect.
```
Comment (plaindocs) [inline docs/conf/eu/2017/news/announcing-events-volunteers.rst]: <3

**Comment by plaindocs** [comment]
LGTM

**Pull request merged**